routes/races/races.py

```diff
@@ -14,8 +14,8 @@ def races(round_id):
 
     races = Race().get_races_by_round(round_id)
     current_round_toggle = validate_current_round_not_started(round_id)
-
-    return render_template('races.html', races=races, round_id=round_id, current_round_toggle=current_round_toggle)
+    now = time_now()
```

Comment:
This seems odd. Can we just return datetime.now? or at least time_now(). There's no need to assign variables for bants.

Reply:
This has been addressed. Thank you for your feedback.